src/map.rs

```diff
@@ -277,10 +315,25 @@ where
     }
 
     #[inline]
-    /// Tests if `key` is a key in this table.
+    /// Returns `true` if the map contains a value for the specified key.
     ///
-    /// The key may be any borrowed form of the map's key type, but `Hash` and `Eq` on the borrowed
-    /// form must match those for the key type.
+    /// The key may be any borrowed form of the map's key type, but
+    /// [`Hash`] and [`Eq`] on the borrowed form *must* match those for
+    /// the key type.
+    ///
+    /// [`Eq`]: std::cmp::Eq
+    /// [`Hash`]: std::hash::Hash
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use flurry::HashMap;
+    ///
+    /// let map = HashMap::new();
+    /// map.pin().insert(1, "a");
+    /// assert_eq!(map.pin().contains_key(&1), true);
+    /// assert_eq!(map.pin().contains_key(&2), false);
```

Comment:
This may be a good place to demonstrate that you can re-use `map.pin()`:
```rust
let mref = map.pin();
mref.insert(1, "a");
assert_eq!(mref.contains_key(&1), true);
assert_eq!(mref.contains_key(&2), false);
```